package/yast2-add-on.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Mon Dec  5 17:37:00 UTC 2022 - Ladislav Slezák <lslezak@suse.cz>
+
+- Fixed failure with CDATA URL containing white space in AutoYaST
```

Comment:
Yes, but importantly, we only fix the `media_url` element and we should mention that.
When I check the schema, it turns out that it is the only URL kind handled in this package (right?) but that is not obvious at first sight.